src/hotspot/share/runtime/globals.hpp

```diff
@@ -569,6 +569,9 @@ const int ObjectAlignmentInBytes = 8;
   product(bool, PrintNMTStatistics, false, DIAGNOSTIC,                      \
           "Print native memory tracking summary data if it is on")          \
                                                                             \
+  product(bool, DoEcoreOpt, false, DIAGNOSTIC,                              \
+          "Perform Ecore Optimization")                                     \
+                                                                            \
```

Comment:
I think this should be a CPU specific flag in ./cpu/x86/globals_x86.hpp (similar to how we have linux specific flags in ./os/linux/globals_linux.hpp). Also the description should clarify that the default is actually true for Ecore systems, and false elsewhere.

Reply:
Thanks, didn't know about that file, will try.